src/connections/api.js

```diff
@@ -0,0 +1,193 @@
+require('dotenv').config(); // Load environment variables from .env
+const { onTwitchMessage } = require('../modules/legacy');
+const Logger = require('../utils/logger');
+const WebSocket = require('ws');
+const ReconnectingWebSocket = require('reconnecting-websocket');
+const jwt = require('jsonwebtoken');
+const NodeCache = require('node-cache'); // Import node-cache
```

Comment:
I think these are missing in the package.json

Reply:
I asked space to npm install the modules his side.

Reply:
All dependencies should be in `package.json`, that's how NPM projects work. You can then just `npm ci` or `npm install` and get the expected dependencies needed for the project.

Reply:
does running npm install not add them to the package.json?

Reply:
Yes, `npm install <dep>` should add them to package.json, so I'm surprised you haven't already included them as part of this PR